stock_picking_batch_extended/wizard/stock_picking_to_batch.py

```diff
@@ -1,4 +1,5 @@
 # Copyright 2012-2016 Camptocamp SA
+# Copyright 2023 FactorLibre - Boris Alias
```

Comment:
No change in this file, so remove this copyright

Reply:
It has already been deleted